tsl/src/gapfill/planner.c

```diff
@@ -154,7 +155,7 @@ gapfill_plan_create(PlannerInfo *root, RelOptInfo *rel, struct CustomPath *path,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 				 errmsg("gapfill functionality with window functions not supported")));
 
-	for (int i = 0; i < list_length(tlist); i++)
+	for (i = 0; i < list_length(tlist); i++)
```

Comment:
Line 178 is also a problem

Reply:
can you give me the CFLAGS you are compiling with?

Reply:
or the output of `pg_config --cflags`

Reply:
Unfortunately not easily. But I'm not passing anything extra on a Debian 8 instance

Reply:
yea but we add postgres cflags to our own so it must come from those

Reply:
Here's the result from installing PG11 on a Debian 8 machine:
```
# pg_config --cflags
-Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fPIC -pie -fno-omit-frame-pointer
```